src/main/java/oshi/software/os/linux/LinuxFileSystem.java

```diff
@@ -71,10 +114,19 @@
             // FileStore toString starts with path, then a space, then name in
             // parentheses e.g., "/ (/dev/sda1)" and "/proc (proc)"
             String path = store.toString().replace(" (" + store.name() + ")", "");
-            // Exclude special directories
-            if (path.startsWith("/proc") || path.startsWith("/sys") || path.startsWith("/run") || path.equals("/dev")
-                    || path.equals("/dev/pts"))
-                continue;
+
+            // Exclude pseudo file systems
+            if (this.pseudofs.contains(store.name())) {

```

Comment:
My default position is just to use the filesystems that Java returns.  I figure whoever wrote those did all the hard thought work required.   I generally would prefer to return more information to OSHI users and let them do the filtering on their end, rather than arbitrarily choose how to filter.

That said, I think adding the mount point field is a good idea.  I think OSHI already parses that internally to get the path, but we don't display that.